### PR TITLE
Mimir 1135 bestbet innholdstype og emne oversettes

### PR DESCRIPTION
**Pull request by Glenruben**

### CODE REVIEW

File: src/main/resources/react4xp/_entries/bestbet/Bestbet.jsx [hunk 1]
```diff
@@ -145,6 +148,7 @@ function Bestbet(props) {
         startDateValue: item.linkedContentDate,
         contentTypeValue: item.linkedContentType,
         mainSubjectValue: item.linkedContentSubject,
+        englishMainSubjectValue: item.englishMainSubjectValue,
```
Comment (johnnadeluy) [inline src/main/resources/react4xp/_entries/bestbet/Bestbet.jsx:151]: Should correspond with the repo properties: linkedEnglishContentSubject.

File: src/main/resources/services/bestBetList/bestBetList.es6 [hunk 2]
```diff
@@ -29,6 +30,9 @@ exports.get = () => {
 }
 exports.post = (req) => {
   const body = JSON.parse(req.body)
+
+  log.info('GLNRBN logger body: ' + JSON.stringify(body, null, 2))
```
[the remark this answers is not in the document]
Comment (Glenruben) [reply src/main/resources/services/bestBetList/bestBetList.es6:34]: 🪵

File: src/main/resources/site/parts/searchResult/searchResult.ts [hunk 3]
```diff
@@ -146,9 +146,12 @@ export function renderPart(req: Request): React4xpResponse {
...
         title: title,
         preface: firstBet.data.linkedContentIngress ? firstBet.data.linkedContentIngress : '',
-        contentType: firstBet.data.linkedContentType ? firstBet.data.linkedContentType : '',
+        contentType: firstBet.data.linkedContentType ? localize({
+          key: `contentType.search.${firstBet.data.linkedContentType}`,
+          locale: language
+        }) : '',
         url: href,
-        mainSubject: firstBet.data.linkedContentSubject ? firstBet.data.linkedContentSubject : '',
+        mainSubject: firstBet.data.linkedContentSubject ? getSubjectForLanguage(firstBet) : '',
```
Comment (johnnadeluy) [inline src/main/resources/site/parts/searchResult/searchResult.ts:154]: Should firstBet.data.linkedEnglishContentSubject be included in the conditional?

**Review by johnnadeluy — APPROVED** [review]
Great job! Clever workaround and the code is neat as always. ✨

File: src/main/resources/site/parts/searchResult/searchResult.ts (hunk 3 above)
Comment (Glenruben) [reply src/main/resources/site/parts/searchResult/searchResult.ts:154]: Don't think it's necessary. See the function getSubjectForLanguage, it checks if the english field exists. If it does not, it returns the default field, which is checked here.

Comment (Glenruben) [reply src/main/resources/site/parts/searchResult/searchResult.ts:154]: Added a comment in the code.

**Pull request merged**